src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2356,6 +2356,28 @@ struct AttentionExecutor : public PagedAttentionExecutor {
                 _slot_mapping.ptr<int32_t>()[idx++] =
                     block_number * _helper._block_size + block_offset % _helper._block_size;
             }
+            // To simplify tails of the kernels for Q*K and W*V:
```

Comment:
this is a WA for first-token-kernels which couldn't correctly support tails, `matmul(attn_socre, value)` to be exact, why this WA not near the code of kernels?

Reply:
It will be useful if padding zero logic is merged into `exec_loop_mixed/pack kv`, but if merged the workitem will be reduced from header number*zero tokens to header number. So keep it here still reasonable since here is the centralized logic to handle the destination kvcache.